common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1430,6 +1430,8 @@
     // account listener rollback handler on failure
     public static final KnownKey rollback_on_account_listener_failure = KnownKey.newKey(true);
 
+    public static final KnownKey attachment_extraction_client_class = KnownKey.newKey("com.zimbra.cs.convert.LegacyConverterClient");
```

Comment:
can you please add comment what is value of the key if one need to switch to tika?